src/components/OrderingPage/OrderingPart.tsx

```diff
@@ -0,0 +1,83 @@
+import './OrderingPart.scss';
+import { OrderingForm } from '../OrderingForm/OrderingForm';
+import { useState } from 'react';
+import x from './x.svg';
+
+export const OrderingPart = () => {
+  const [count, setCount] = useState(1);
+  const incrementCount = () => {
+    setCount(count + 1);
+  };
+
+  const [{ items }, setItems] = useState({ items: [<></>] });
+  const addItem = () => {
+    items.push(
```

Comment:
you do it slightly wrong here)
you need to create a component for the html below and you have to store only data (an array of objects) in the component's state